recipes/nyuki_recipe/build.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+$PYTHON setup.py install
+
+# Add more build steps here, if they are necessary.
+
+# See
+# http://docs.continuum.io/conda/build.html
+# for a list of environment variables that are set during the build process.
```

Comment:
Please remove comments

Reply:
Certainly. I will remove all the comments.Thanks @scopatz .

Reply:
@scopatz Yep, I removed these comments.